### ISSUE TITLE
`__dirname` is not supported by cloudflare pages

### ISSUE THREAD

**Issue by StefanH-AT** [harlan-zw/nuxt-og-image#93]
### Describe the bug

I'm trying to deploy a site on cloudflare pages and this module breaks because it's trying to resolve `__dirname` which is not supported by cloudflare pages

**Comment by DanSnow** [comment]
I've also encountered this issue. Upon further investigation, I found that the `__dirname` variable appears in the [css-inline](https://www.npmjs.com/package/css-inline) package

<img width="784" alt="image" src="https://github.com/harlan-zw/nuxt-og-image/assets/5575082/41958653-636e-4c09-8b67-5154c39469ea">

**Comment by harlan-zw** [comment]
Hi, looking into a fix for this. In the meantime, you can disable the `cssInline` or use version 2.0.x.

```ts
export default defineNuxtConfig({
  runtimeCompatibility: {
    cssInline: false
  }
})
```

**Comment by harlan-zw** [comment]
Hi, please use 2.1.2

**Comment by StefanH-AT** [comment]
Never used `runtimeCompatibility` but it's showing errors in my ide and doesn't fix the issue either. I had to downgrade to 2.0.x

**Comment by DanSnow** [comment]
Hi @harlan-zw,

I tried version 2.1.2, but there are still issues. The problem occurs whenever we load css-inline. Simply disabling css-inline doesn't seem to be a viable solution.

Additionally, I've left a comment on the css-inline side. I've tested using wasm-pack, and on the Nuxt side, I had to make modifications to `nuxt.config.ts` like this to load the re-packaged `css-inline` without encountering errors.

```ts
import wasm from '@rollup/plugin-wasm'

defineNuxtConfig({
  nitro: {
    rollupConfig: {
      plugins: [wasm()],
    },
  },
})
```

However, I haven't tested whether `css-inline` is functioning correctly.

**Comment by harlan-zw** [comment]
Thanks for your help @DanSnow, I think there was a bug in disabling the cssInline compatibility for non-node environments. That should be fixed in 2.1.3.

If you're using Node you may have to provide the following:

```ts
ogImage: {
    runtimeCompatibility: {
      cssInline: false,
    },
}
```

Will continue to debug this and hopefully we can get it working for all environments soon.

**Comment by harlan-zw** [comment]
The core issue here is solved with the above workaround and should just work in v3 of the module.

I have made a new issue to track supporting `css-inline` in cloudflare pages https://github.com/harlan-zw/nuxt-og-image/issues/119
